chord_metadata_service/discovery/fields.py

```diff
@@ -189,18 +166,31 @@ async def get_month_date_range(field_props: DiscoveryFieldProps) -> tuple[str |
 
 
 async def get_range_stats(field_props: DiscoveryFieldProps, low_counts_censored: bool = True) -> list[BinWithValue]:
-    model, field = get_model_and_field(field_props["mapping"])
+    model, field = f_utils.get_model_and_field(field_props["mapping"])
+
+    # JSONField specific
+    group_by = field_props.get("group_by")
+    group_by_value = field_props.get("group_by_value")
+    value_mapping = field_props.get("value_mapping")
 
     # Generate a list of When conditions that return a label for the given bin.
     # This is equivalent to an SQL CASE statement.
-    whens = [
-        When(
-            **{f"{field}__gte": floor} if floor is not None else {},
-            **{f"{field}__lt": ceil} if ceil is not None else {},
-            then=Value(label),
-        )
-        for floor, ceil, label in f_utils.labelled_range_generator(field_props)
-    ]
+    if group_by and group_by_value and value_mapping:
+        # JSONField array specific
```

Comment:
reading this code, it isn't immediately clear why it applies to just JSON fields. I guess it's because `group_by` is only specified as a separate prop in the config to drill into JSON fields (maybe in search v2 this won't exist then, since we can use the JSON schema to detect if we're drilling into a JSON field)? maybe add a comment clarifying